be/src/runtime/exec_env.cpp

```diff
@@ -236,6 +236,13 @@ Status ExecEnv::_init(const std::vector<StorePath>& store_paths) {
     int connector_num_io_threads = config::pipeline_connector_scan_thread_num_per_cpu * CpuInfo::num_cores();
     CHECK_GT(connector_num_io_threads, 0) << "pipeline_connector_scan_thread_num_per_cpu should greater than 0";
 
+    if (config::hdfs_client_enable_hedged_read) {
+        // Set hdfs client hedged read pool size
+        config::hdfs_client_hedged_read_threadpool_size = std::min(connector_num_io_threads * 2, 128);
```

Comment:
why change this value?

Reply:
Just make the threadpool size change dynamic with node's cpu

Reply:
I see, but it's very weird to change the config in this way. 

maybe `hdfs_client_hedged_read_threadpool_max_size`

and
```
config: hdfs_client_hedged_read_threadpool_max_size = min(2 * connector, hdfs_client_hedged_read_threadpool_max_size) 
```